sql-parser/src/main/antlr/SqlBase.g4

```diff
@@ -755,6 +757,8 @@ ALWAYS: 'ALWAYS';
 
 READ: 'READ';
 
+USER: 'USER';
```

Comment:
is `USER` an new reserved keyword? if so, it needs to be documented in breaking changes